edx_proctoring/apps.py

```diff
@@ -5,11 +5,11 @@
 
 from __future__ import absolute_import
 
-import json
 import os
 import os.path
 import warnings
 
+import simplejson as json
```

Comment:
Did you hit a problem that required adding this, or is it just to reduce the odds of encountering a bytes vs. str issue in production?  We're currently on track to have edx-platform on Python 3 in production tomorrow, so not sure it makes sense to make more proactive changes of that sort.